frontend/src/components/applications/CancelApplicationDialog.vue

```diff
@@ -60,6 +60,9 @@ export default {
       },
     },
   },
+  created() {
```

Comment:
Is there any reason we have a constant and a hard-coded default in applicationType? If it's only used in applicationType there's no need for a constant. If it's used elsewhere then applicationType should use the constant.

Reply:
Meant to get rid of the constant. Fixed.